scitt/create_detatched_signed_statement.py

```diff
@@ -0,0 +1,217 @@
+""" Module for creating a SCITT signed statement with a detached payload"""
+
+import hashlib
+import argparse
+
+from typing import Optional
+
+from hashlib import sha256
+
+from pycose.messages import Sign1Message
+from pycose.headers import Algorithm, KID, ContentType
+from pycose.algorithms import Es256
+from pycose.keys.curves import P256
+from pycose.keys.keyparam import KpKty, EC2KpD, EC2KpX, EC2KpY, KpKeyOps, EC2KpCurve
+from pycose.keys.keytype import KtyEC2
+from pycose.keys.keyops import SignOp, VerifyOp
+from pycose.keys import CoseKey
+
+from ecdsa import SigningKey, VerifyingKey
+
+
+# Feed header label comes from version 2 of the scitt architecture document
+# https://www.ietf.org/archive/id/draft-birkholz-scitt-architecture-02.html#name-envelope-and-claim-format
+HEADER_LABEL_FEED = 392
```

Comment:
Do we nee this now that we have SUB in the cwt_claims?

Reply:
good question, yes we do, our backend has not moved over fully to sub instead of feed. Sub is just lifted and put on event attributes. 

Feed is used as the value to based the asset search on.

Reply:
removed now as we use cwt.subject in the backend from next release